app/experimenter/experiments/tests/test_changelog_utils.py

```diff
@@ -30,14 +32,20 @@ def setUpClass(cls):
 
     def test_outputs_expected_schema_for_empty_experiment(self):
         owner = UserFactory.create()
-        experiment = NimbusExperiment.objects.create(owner=owner)
+        updated_time = timezone.datetime(
+            year=2022, month=1, day=2, hour=0, minute=0, second=0
+        )
+        with mock.patch("django.utils.timezone.now") as mock_now:
+            mock_now.return_value = updated_time
+            experiment = NimbusExperiment.objects.create(owner=owner)
```

Comment:
Oh sorry I wasn't focusing on this earlier, I don't think you need to go through all the trouble of mocking out now, you can just use `experiment._updated_date_time.strftime(....)` for whatever the value is here.  We're not explicitly testing the saving behaviour here, just that whatever is in the db is serialized.

Reply:
oh so if I don't add the field we are getting that field in the test case, so that's why I mocked the field